src/WeatherWidget/WeatherWidget.js

```diff
@@ -1,62 +1,50 @@
 import './WeatherWidget.scss';
-import cloudy from '../assets/cloudy.svg';
+import HourWeather from "../HourWeather/HourWeather";
+import { useState, useEffect } from "react";
+import weatherIcons from "./icons.js";
+import Loading from "../Loading/Loading";
 
 function WeatherWidget() {
+  const [forecast, setForecast] = useState(null);
+
+  useEffect(() => {
+    fetch("https://aqueous-savannah-80171.herokuapp.com/forecasts?location=denver, co")
+    .then(response => response.json())
+    .then(data => {console.log(data) 
+      setForecast(data.data.attributes)})
+    .catch(error => console.log(error))
+  },[])
+
   return ( 
+  <div>
```

Comment:
We might have to shift some of the styling for `.weather-widget` to this div so it occupies the correct amount of space. Maybe just `grid-column: 1 / 3`?